nilearn/plotting/find_cuts.py

```diff
@@ -209,6 +210,13 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
 
     data = _smooth_array(data, affine, fwhm='fast')
 
+    if (isinstance(n_cuts, numbers.Number) and n_cuts > 0):
```

Comment:
Not 100 % convinced I have to say. At the very least I would advocate strict equality checking, i.e. round(n_cuts) == n_cuts.